contracts/keep3r/Keep3rAbstract.sol

```diff
@@ -33,4 +43,39 @@ contract Keep3r {
     _;
     keep3r.worked(msg.sender);
   }
-}
\ No newline at end of file
+  // Checks if caller is a valid keeper, handles payment amount after execution
+  modifier paysKeeperAmount(uint256 _amount) {
+    _isKeeper();
+    _;
+    keep3r.workReceipt(msg.sender, _amount);
+  }
+  // Checks if caller is a valid keeper, handles payment amount in _credit after execution
+  modifier paysKeeperCredit(address _credit, uint256 _amount) {
+    _isKeeper();
+    _;
+    keep3r.receipt(_credit, msg.sender, _amount);
+  }
+  // Checks if caller is a valid keeper, handles payment amount in ETH after execution
+  modifier paysKeeperEth(uint256 _amount) {
```

Comment:
don't see these modifiers used on the code, is that correct?

Reply:
Yup, they are not being used, since we use the default payments for the current jobs.